spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala

```diff
@@ -74,7 +74,7 @@ trait ConformanceExecution extends CommonJobExecution {
 
     // Enable Menas plugin for Control Framework
     MenasPlugin.enableMenas(
-      conf,
+      conf.config,
```

Comment:
this looks quite misleading, therefore the ConfigReader probably should be renamed to e.g. `confReader`?